src/Security/Member.php

```diff
@@ -1044,7 +924,7 @@ public function onAfterWrite()
 
         Permission::reset();
 
-        if ($this->isChanged('Password')) {
+        if ($this->isChanged('Password') && static::config()->get('password_logging_enabled')) {
```

Comment:
Added feature to disable password logging as it could be seen as a security risk to store previous passwords.